recipes/crypten/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "crypten" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: f9d76bc0ba1468cb77261b8d5486ee875266363310353a18aba948429aa24fda
+  - url: https://raw.githubusercontent.com/facebookresearch/CrypTen/main/requirements.txt
+    sha256: 14569c02e2bb5233d9d75a13646a7e9eaee0a1bcc8e7ffe4044ad9a3511d8eca
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - setuptools
+    - pytest-runner
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+    - pytest-runner
+  run:
+    - future
+    - setuptools
+    - omegaconf >=2.0.6
+    - onnx >=1.7.0
+    - pandas >=1.2.2
+    - python >=3.6
+    - pyyaml >=5.3.1
+    - scipy >=1.6.0
+    - scikit-learn
+    - tensorboard
+    - pytorch >=1.7.0
+    - torchvision >=0.9.1
+    - pytest-runner
+
+test:
+  imports:
+    - crypten
+  requires:
+    - pip
+
+about:
+  home: https://github.com/facebookresearch/CrypTen
+  summary: 'CrypTen: secure machine learning in PyTorch.'
+  license: MIT
+  license_file: LICENSE
```

```suggestion
  license_family: MIT
  license_file: LICENSE
```
Could you please make a PR asking upstream to add the package to the pypi source.

They can do so like:
https://github.com/scikit-image/scikit-image/blob/main/setup.cfg#L10

When you do, please cross reference the issue here so you can cleanup in future versions.